CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [3.0.0] - 2025-01-31
+
+### Changed
+
+- Updated XE endpoint_url to use an alternative one as per their recommendation.
+- Added dependencies and updated requires that were using ruby's standard library no longer supported since Ruby 3.0.0
+- Updated workflow to account for different ruby versions.
+- Updated minimum required ruby version as per the newly added workflow.
```

Comment:
Can probably get rid of these two lines, they aren't relevant to consumers. I think we should mention what Ruby versions the new release will support. https://www.ruby-lang.org/en/downloads/branches/ has a detailed maintenance timeline for all supported Ruby versions.

Reply:
Done in [03b05f7](https://github.com/degica/omni_exchange/pull/9/commits/03b05f766958282f9728d8ac00239447eaf1b0aa).